node-notes-cli/update.js

```diff
@@ -0,0 +1,18 @@
+import * as fs from 'node:fs/promises';
+import readNotes from './read.js';
+
+export default async function updateNote(id, notes) {
+  try {
+    const dataCopyObject = await readNotes(0);
+    if (dataCopyObject.notes[id]) {
+      dataCopyObject.notes[id] = notes.join(' ');
```

Comment:
You did this `.join()` above. The intent of the CLI, though, is to have just one command line argument that is the note, wrapped in quotes. That's the most common way to write a CLI.